build.gradle

```diff
@@ -54,17 +55,17 @@ compileJava {
 }
 
 dependencies {
-    compile('org.seleniumhq.selenium:selenium-java:3.4.0'){
+    compile('org.seleniumhq.selenium:selenium-java:3.5.1'){
```

Comment:
Seems like this might be a good match to our purposes: https://stackoverflow.com/questions/39664589/use-semantic-versioning-for-a-dependency-in-gradle